examples/README.md

```diff
@@ -27,5 +27,102 @@ cargo run --all-features --bin EXAMPLE-NAME
 
 Please be sure to have installed all the required libraries before building examples (the list is available on the [gtk-rs](https://github.com/gtk-rs/gtk/) repository).
 
+## Example Screenshots
+
+Screenshots of examples using cario or futures is missing. Examples are from cross compiling projects for Windows 10.
```

Comment:
```suggestion
Screenshots of examples using cairo or futures are missing. Examples are from cross compiling projects for Windows 10.
```

Why does it matter that it's from Windows 10?

Reply:
I thought it would render differently based on what it was targeted for.

Reply:
That's just Adwaita, so unless you configure a different theme this would also be how it looks like (minus font rendering differences) on a default GNOME setup :)

Reply:
Ah! Thank you, maybe it should say Adwaita instead of Windows.